bigflow/workflow.py

```diff
@@ -104,6 +104,7 @@ class Job(abc.ABC):
     id: str
     retries: int = 3
     retry_delay: float = 60
+    job_execution_timeout: int
```

Comment:
Add default value for `job_execution_timeout` here.
Otherwise attribute is missing, (type hint doesn't declare actual attribute).

Reply:
Done